### ISSUE TITLE
Pressure showing on app but not on M5 screen

### ISSUE THREAD

**Issue by Skallagrimmm** [WeatherXM/issue-tracker#186]
## WeatherXM Related Element
Please pick one of the following WeatherXM elements that are applicable for this bug:

- Android Application

- M5

## **Describe the bug**
Pressure is displayed as (--) on the M5 screen but displayed properly on Android app at same time.

### **How To Reproduce**

Compare pressure readings between M5 and app for station Smooth Pastel Degrees 

### **Expected behavior**
Both show same readings.

### **Screenshots**
![Screenshot_20230315-185303~2.png](https://user-images.githubusercontent.com/100298772/225490671-ded149d2-84ce-40a9-934f-3308075eafce.png)

![PXL_20230316_015315459.jpg](https://user-images.githubusercontent.com/100298772/225490676-c206e030-694c-460e-af1e-a9b293b14512.jpg)

### **If Applicable: Smartphone (please complete the following information that are needed for this bug):**
 - Device: Google Pixel 6a
 - OS: [android 13]
 
 - Android App Version 3.3.3-7a5f0a0
 

### **If Applicable: Hardware (please complete the following information):**
 - Firmware Version 0.4.0

**Comment by Skallagrimmm** [comment]
Historical charts show the pressure readings stopped recording hours ago...while all other readings were fine..
Problem with the sensor in the M5?
![Screenshot_20230315-211412.png](https://user-images.githubusercontent.com/100298772/225512799-0621c7e0-56d0-4de7-bf41-f77e1377b950.png)

**Comment by liliumjsn** [comment]
Does the problems remain after resetting your M5?

**Comment by liliumjsn** [comment]
Have you dropped the device or loosen the 4 stack screws? 
The pressure sensor is located into the GPS stack.

Can you please:
1. Turn off your device
2. Loosen the 4 stack screws on the bottom
3. Disassemble and reassemble the stacks
4. Tight back the screws
5. Turn on device

There is a video guide for switching to the external antenna, 
but you can just follow the steps for disassembling the stacks.
https://www.youtube.com/watch?v=FP4rT3FTz0M

**Comment by Skallagrimmm** [comment]
Yep, looks like this did the trick. It's now reading pressure and all the other statistics just fine.

**Comment by Greg15** [comment]
Does this reveal a different bug in the app or API then?
If I understand correctly, the chart is indicating that no pressure reading was provided for hours, but the "current weather" conditions in the app was showing the last provided but entirely outdated pressure reading, rather than properly trapping the condition that their was no current pressure reading available.

Looks like the chart and M5 handled it properly, but the "current conditions" part of the app did not.

Thoughts?

**Comment by Skallagrimmm** [comment]
@liliumjsn I spoke too soon. The chart has no pressure data recorded since the M5 was put back together and restarted.

I'm not onsite and it's the middle of the night. I'll update with the host in the morning and get back to you about what the M5 is doing now.

**Comment by Skallagrimmm** [comment]
@liliumjsn @NikosTsiligaridis 

Hey guys, I'm having some issues again with the same M5

The M5 screen is just white, There is no data on the app and it reports as being offline on the app

I opened up the M5, took the stacks apart, switch the stacks blue and gray and put them back together

The screen briefly appeared showed all data except for the pressure again, which was the problem in the first place with this station

It then flashed and went white again,

I again took the stacks apart, put them back to their original orientation and closes up again. But it still just remained white. Tried to power on and off and reset. Nothing worked just white screen.

Any ideas?
